bin/BuildPackages.sh

```diff
@@ -279,8 +248,8 @@ build_one_package() {
   ) || build_fail
 }
 
-date >> "$LOGDIR/$FAILPKGFILE.log"
-for PKG in "${PACKAGES[@]}"
+date >> "$LOGDIR/fail.log"
+for PKG in ${PACKAGES}
```

Comment:
The advantage of `PACKAGES+=("$1")` and `for PKG in "${PACKAGES[@]}"` is that it will support directory names with spaces in.